### DISCUSSION TITLE
Reuse deleted account's name?

### DISCUSSION THREAD

**Discussion by Lampe2020** [community/community#125803] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have two accounts, @Lampe2020 and @Stehlampe2020. I have created @Lampe2020 later on because of... reasons? (I don't know why I didn't just change my account username)   
And now I want to merge both accounts but have activity on both. It's not too hard, as there's even a guide in the GitHub docs on how to do it. The only concern I have is that my older activity is on @Stehlampe2020 but I want the username @Lampe2020. So can I merge them into @Stehlampe2020, delete @Lampe2020 and then change my username to @Lampe2020 so I have my badges and old activity still intact but on the new username?   

My concern is that the username @Lampe2020 may become unavailable after I delete that account, so is there any reason to fear that or will that not happen?

**Comment by ghost** [comment]
Hello there @Stehlampe2020!

After you delete your account, your username will be unavailable to anyone for 90 days or 3 months, after which you can claim your username. I would not worry about your username being used by someone else, but it depends.

Source: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/deleting-your-personal-account

**Reply by ghost** [reply]
To avoid any confusion, I wanted to state that I am not part of the GitHub staff team.

Onwards to your question, there is no way to transfer *GitHub commit history* to the extent of my knowledge, but you can transfer repositories and then commit to them in your new account.

**Reply by Ahmed8881** [reply]
After deleting your account, your username will be unavailable to anyone for 90 days (or 3 months). After this period, you can reclaim your username.

**Reply by ebndev** [reply]
Hi @YudhJT, thanks for taking the time to answer!

So you are aware, we made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

**Comment by raynepaws** [comment, marked as answer]
i would suggest changing @Lampe2020 to something else, like a jumble of random letters, and then deleting it. this way, you can claim @Lampe2020 right after deletion. :)

**Reply by Lampe2020** [reply]
I now did a little switcheroo, Lampe2020 first became Lampe2020-tmp, then Stehlampe2020 became Lampe2020, now Lampe2020-tmp is Stehlampe2020.
